include/psp2/motion.h

```diff
@@ -14,50 +14,239 @@
 extern "C" {
 #endif
 
+/* enum */
+
+typedef enum SceMotionMagFieldStability {
+	SCE_MOTION_MAGFIELD_UNSTABLE = 0,
+	SCE_MOTION_MAGFIELD_UNUSED   = 1,
+	SCE_MOTION_MAGFIELD_STABLE   = 2
+} SceMotionMagFieldStability;
+
+typedef enum SceMotionErrorCode {
+	SCE_MOTION_ERROR_DATA_INVALID     =	0x80360200,
```

Comment:
Can you fix identation here?

Reply:
I am not sure what happened to indentation its fine locally.

Reply:
You're using tabs most likely. For comments, it's better to use spaces to ident.

Reply:
I figured ... guess its fixed now